roaring64/bsi64_test.go

```diff
@@ -455,3 +455,32 @@ func TestMinMaxWithRandom(t *testing.T) {
 	assert.Equal(t, bsi.MinValue, bsi.MinMax(0, MIN, bsi.GetExistenceBitmap()))
 	assert.Equal(t, bsi.MaxValue, bsi.MinMax(0, MAX, bsi.GetExistenceBitmap()))
 }
+
+func TestBSI_WriteTo_ReadFrom(t *testing.T) {
+	file, err := os.CreateTemp("./testdata", "bsi-test")
```

Comment:
There are two issues here. One issue is that some users of the library want to run tests without access to the disk. It is fine to include disk access in the tests, but in case of IO failure, we should not fail the test. The second one is that os.CreateTemp is not part of Go 1.14 and we want to support Go 1.14 for the time being.